types/operations/comparisons/ComparisonID.hpp

```diff
@@ -54,6 +54,28 @@ extern const char *kComparisonShortNames[
     static_cast<typename std::underlying_type<ComparisonID>::type>(
         ComparisonID::kNumComparisonIDs)];
 
+/**
+ * Flips a comparison. As in greater than flips to less than, less than flips
+ * to greater than, and similarly for greater/less than or equals. Notice that
+ * flipping equals results in equals, same for not equals.
+ * @param comparison - The Id of a comparison to flip.
```

Comment:
Add an extra line between the prose text and the `@param`/`@return` descriptions.